method-README-template.md

```diff
@@ -1,65 +1,71 @@
 
 ## Description
-- Provide a brief and clear description of the method, its purpose, and what it aims to achieve. Add a link to a related paper from social science 
-domain and show how your method can be applied to solve that research question.  
-For example,
-4TCT is a specialized tool designed for the efficient collection of textual data from the 4chan platform. It automates the process of gathering posts from various boards, aiming to facilitate research and analysis in social science and computational linguistics.
-This tool is particularly useful for analyzing online discourse, community dynamics, and trends within the 4chan ecosystem. It can support studies on topics like meme culture, information dissemination, and the impact of anonymous social media on public opinion.
+1. Provide a brief and exact description of the method clearly mentioning its purpose i.e., what the method does or aims to achieve in abstract terms (avoiding technical details).
+2. Focus should be to emphasize on the functionality as a blackbox, helping researchers with different levels of expertise and experience to understand what the method is doing.
+3. Brielfly explain the input and output of the method and its note worthy features.
+4. Provide link(s) to related papers from the social science domain using the method or similar methods for solving social science research questions. 
+5. In a separate paragrpah, highlight the reproducibility aspect of the method providing details or references to the resources used by the method, the data used in building the pre-trained modules etc.
+6. It should also provide different configuration parameters to assist in altering the behavior of the method for a related use case. 
 
-## Keywords
-Few keywords placed after description highlighting the nature of the method 
 
-## Science Usecase(s)
-- Include usecases from social sciences that would make this method applicable in a certain scenario. 
-The use cases or research questions mentioned should arise from the latest social science literature cited in the description.
-For example,
-How to collect 4chan data from on political discourse from dates ..
- 
+## Keywords
+- Keyword 1 (3 to 5 keywords highlighting the purpose of method)
+- Keyword 2
+- Keyword 3   
 
-## Repo Structure
-- Explain the overall structure of the method, including directories, key files, and their functions.
-For example,
-The tool's architecture includes a src/ directory for core scripts, with requester.py handling data collection, board.py managing board-specific requests, and utils.py for auxiliary functions. Data is stored in a data/ directory created upon initiation, and documentation is available in docs/.
+## Social Science Use Case(s)
+1. Include use case(s) or research question(s) from social sciences that would make this method applicable in a certain scenario.
+2. They should arise from the latest social science literature cited in the description. 
 
+## Git Repository Structure
```

Comment:
My opinion is that this section should be optional. If the method is a Python or R package, the Git repository structure is already standardised and no explanation should be given. For the case that the method is a Jupyter Notebook, this section should be mandatory.

@chainsawriot do you agree with me?

The Git repository structure of a Python package is illustrated at "[src layout vs flat layout](https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/)" from "[Python Packaging User Guide](https://packaging.python.org/en/latest/)".

The Git repository structure of a R package is illustrated at "[Package structure and state](https://r-pkgs.org/structure.html)" from "[R Packages (2e)](https://r-pkgs.org/)".

Reply:
I think this is optional. Perhaps it is only applicable to unpackaged software. And we surely sure encourage packaged software to enable reusability.